packages/vscode-extension/src/commonlib/common/constant.ts

```diff
@@ -9,3 +9,10 @@ export const signingIn = "SigningIn";
 export const loggedOut = "LoggedOut";
 export const loggedIn = "LoggedIn";
 export const loggingIn = "LoggingIn";
+
+export const subscriptionInfoFile = "subscriptionInfo.json";
+export const envDefaultJsonFile = "env.default.json";
+export const solution = "solution";
+export const subscriptionIdString = "subscriptionId";
```

Comment:
No need to define constants for object property name. Instead of:

``` typescript
const xxxString = "xxx";
const x = object[xxxString];
```

Just use:
``` typescript
const x = object.xxx;
```

Reply:
Fixed.